v2/AdminCore.Services/Configuration/DependencyInjection.cs

```diff
@@ -0,0 +1,97 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="DependencyInjection.cs" company="Admincore">
```

Comment:
If we're Open Sourcing all of this should we be copyrighting all these files for the company "Unosquare"?